microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools-100.md

```diff
@@ -0,0 +1,136 @@
+
+
+<!-- ====================================================================== -->
+## Join the Microsoft Edge DevTools community on GitHub
+
+<!-- Title: Head to the new DevTools repo on GitHub to send ideas, feedback, suggestions, and bugs -->
+<!-- Subtitle: You can file feedback, ask questions, and have discussions about DevTools in our GitHub repo. -->
+
+We're excited to announce the new [MicrosoftEdge/DevTools: Feedback and discussions about Microsoft Edge Developer Tools (github.com)](https://github.com/MicrosoftEdge/DevTools) repo on GitHub!  This new repo is a place for our community of developers to send us ideas, feedback, suggestions, and tips to improve DevTools. We're eager to discuss with you how DevTools can better serve your needs and meet the challenges you face every day in building great web experiences.
+
+![The DevTools feedback repo.](devtools-100-images/devtools-feedback-repo.png)
+
+Head to [our repo on GitHub](https://github.com/MicrosoftEdge/DevTools) to become a part of the Edge DevTools community.
+
+
+<!-- ====================================================================== -->
+## Filter heap snapshots summary by node type
+
+<!-- Title: Use new filters to focus on specific parts of a heap snapshot -->
+<!-- Subtitle: You can now filter by node type if, for example, you're only interested in the arrays or strings from the heap. -->
+
+When looking at all the objects in a heap snapshot in the **Memory** tool, it can be difficult to focus on specific objects or retaining paths.  Now in Microsoft Edge 100, you can use the new **Node Types** filter when looking at a heap snapshot, to focus on only specific types of nodes.  For example, to see only the arrays and string objects that are in the heap, select the **Array** and **String** entries in the **Node Types** filter.
+
+![Node Types in a heap snapshot in the Memory tool.](devtools-100-images/node-types-heap-snapshot.png)
+
+To learn more about taking a heap snapshot and analyzing the heap in DevTools, see [Record heap snapshots using the Memory tool](../../../memory-problems/heap-snapshots.md).
+
+
+<!-- ====================================================================== -->
+## Use DevTools in Czech and Vietnamese
+
+<!-- Title: DevTools: Now available in Czech and Vietnamese -->
+<!-- Subtitle: Try out DevTools in your preferred language!  If we haven't supported it, yet, let us know. -->
+
+Customizing and simplifying the developer experience includes making our tools available to everyone, regardless of their preferred language.  We've added two new languages, Czech and Vietnamese, to our list of supported languages, and will keep adding more.  If your preferred language isn't supported in DevTools, let us know!
+
+![DevTools in Czech and Vietnamese.](devtools-100-images/czech-vietnamese.png)
+
+For more information, see [Change DevTools language settings](../../../customize/localization.md).
```

Comment:
Just like you did for heap-snapshots.md, please Czech and Vietnamese to the list of supported languages in customize/localization.md. Follow up with Erica if you have any questions!